dac_calibration_server.py

```diff
@@ -86,15 +86,12 @@ def initServer(self):
         self.IQcalsets = {}
         self.DACcalsets = {}
         print 'loading server settings...',
-        yield self.loadServerSettings()
+        self.loadServerSettings()
```

Comment:
why are you not yielding? `self.loadServerSettings` returns a deferred, so unless you yield errors will go into the ether.